lib/features/auth/screens/login_screen.dart

```diff
@@ -53,6 +54,12 @@ class _LoginScreenState extends State<LoginScreen> {
         }
       },
       builder: (context, state) => Scaffold(
+        appBar: AppBar(
+          toolbarHeight: 0,
+          systemOverlayStyle: const SystemUiOverlayStyle(
```

Comment:
can we add this style definition to the app bar theme in the AppTheme.dart?

Reply:
it this is something we need on all screens oc

Reply:
Yes, added the styling to the AppTheme, but the appbar needs to be explicitly called in the scaffolds for the system overlay to be applied.